resources/views/pages/Public/Join.svelte

```diff
@@ -8,98 +8,135 @@
-                Your age
-            </label>
-            <input
-                class="block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none"
-                id="grid-age"
-                type="text"
-            />
-        </div>
+<form class="h-screen-no-nav border border-gray-700 bg-gray-800 p-3 shadow-md" id="join-form">
+  <div class="text-gray-200 text-center">
+    <h1 class="font-bold lg:text-4xl">JOIN ARCOMM</h1>
+    <p class="mt-6">
+      We're looking for well adjusted people who listen to and respect others. While having experience in our games is welcome, it's in no way required. Aspiring members should make an effort to attend the weekly operations, 
+      be accountable for their actions, and generally have a good sense of humor.
+    </p>
+  </div>
+  <hr class="h-px my-8 bg-gray-200">
+  <div class="-mx-3 mb-6 flex flex-wrap">
+      <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-name">
+              Your name
+          </label>
+          <input
+              class="mb-3 block w-full appearance-none rounded border bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:bg-white focus:outline-none invalid:border-red-500"
+              id="grid-name"
+              type="text"
+              placeholder="Name"
+              required
+          />
+          <p class="text-xs italic text-gray-400">This should be the name you use in game</p>
+      </div>
+      <div class="w-full px-3 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-age">
+              Your age
+          </label>
+          <input
+              class="block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none invalid:border-red-500"
+              id="grid-age"
+              type="text"
+              placeholder="e.g. 27"
+              required
+          />
+      </div>
+  </div>
+  <div class="-mx-3 mb-6 flex flex-wrap">
+      <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-location">
+              Location
+          </label>
+          <input
+              class="mb-3 block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none invalid:border-red-500"
+              id="grid-location"
+              placeholder="Make it as long and as crazy as you'd like."
+              required
+          />
+          <p class="text-xs italic text-gray-400">You can be as specific as you like</p>
```

Comment:
I don't think we need both the placeholder and text below to give the same message - maybe change placeholder to `Location`?

Reply:
Placeholder amended to Location